test/unit/org/apache/cassandra/config/YamlConfigurationLoaderTest.java

```diff
@@ -359,6 +362,36 @@ public void converters()
         assertThat(from("sstable_preemptive_open_interval_in_mb", -2).sstable_preemptive_open_interval).isNull();
     }
 
+    @Test
+    public void process()
```

Comment:
I added this test as I was having issues with simulator and thought could be yaml related... kept as its always best to test our configs 😄